src/components/modal/event-modal.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import ModalHeader from './modal-header';
+import EventsTable from '../events-table';
+
+import { Box, LinearProgress, Modal } from '@mui/material';
+
+const style = {
+    modalStyle: {
+        position: 'absolute',
+        top: '50%',
+        left: '50%',
+        transform: 'translate(-50%, -50%)',
+        width: '70vw',
+        backgroundColor: 'background.paper',
+        border: '2px solid #000',
+        boxShadow: 24,
+        p: 4,
+    },
+    modalContentStyle: {
+        overflow: 'auto',
+        maxHeight: '70vh',
+    },
+};
+
+function EventModal({ open, onClose, isLoadingEvent, taskData }) {
+    return (
+        <Modal open={open} onClose={onClose}>
+            <Box sx={style.modalStyle}>
+                <ModalHeader titleId="events" onClose={onClose} />
+                <EventModalContent
+                    isLoadingEvent={isLoadingEvent}
+                    taskData={taskData}
+                />
+            </Box>
+        </Modal>
+    );
+}
+
+EventModal.propTypes = {
+    open: PropTypes.bool.isRequired,
+    onClose: PropTypes.func.isRequired,
+    isLoadingEvent: PropTypes.bool.isRequired,
+    taskData: PropTypes.object.isRequired,
+};
+
+export default EventModal;
+
+function EventModalContent({ isLoadingEvent, taskData }) {
+    return (
+        <Box sx={style.modalContentStyle}>
+            {isLoadingEvent ? (
+                <LinearProgress />
+            ) : (
+                <EventsTable id="modal-modal-description" taskData={taskData} />
```

Comment:
The modal word is duplicated in the id.

Reply:
This id was already present before and I think it comes from a copy and paste of the MUI doc and it indeed has no use so I will delete it